disaster-recovery.md

```diff
@@ -0,0 +1,168 @@
+
+In the event a validator disappears, their multisig key is lost (or, worse, sold during the  bankruptcy proceedings when someone else buys the HSM. Yes, this has happened in the past). To make sure this does not escalate into the next scenario, the other validators need to remove that signer reasonably quickly (a matter of one or two days) to reset the thresholds.
+
+1. Call `remove_signer` on the [MultisigControl Contract](https://etherscan.io/address/0x9d0707C91C67d598808834b4881348684e92E11e#writeContract) with the affected key
+2. Find a reliable validator to replace this party and agree this validator using appropriate validator governance processes
+3. Call `add_signer` to add a new validator on the [MultisigControl Contract](https://etherscan.io/address/0x9d0707C91C67d598808834b4881348684e92E11e#writeContract) 
+
+
+### Scenario 1.7: Loss of control of MultiSig contract on Ethereum
+
+In the event that t+1 validators lose their ETH key/go bankrupt/disappear without trace, and the multisig contract is not adapted fast enough, we lost control of the contract. As there’s limited assets on three for the time being, Vega can be reborn with a new instance.
+
+1. Stop the chain immediately
+2. Conduct root cause to determine how this happened
+3. Restart with a new instance of the contract
+
+
+## Scenario 2: Incorrect Genesis file configuration
+
+In the event that the genesis configuration is incorrect and this causes a major incident when the network has been started:
+
+
+### Scenario 2.1: If no network events have taken place before the incorrect config is noticed
+
+1. Stop the network in coordination with all validators.
+2. Update the [genesis configuration](https://github.com/vegaprotocol/networks#genesis-config-for-validator)
+3. Restart the network from [checkpoint](https://github.com/vegaprotocol/networks#restore-from-checkpoint). 
+
+
+### Scenario 2.2: If the impact of the incorrect configuration CAN be managed/mitigated with the network running and the parameter CAN be changed via governance
+
+1. Create a network parameter governance proposal:  **One** validator to run the command to create proposal >> `vega wallet command --name="testing-mainnet" --pubkey="<my-public-key>" '{<insert json payload of proposal>}'`
+
+2. Coordinate between validators to vote and enact this change:  **All** validators run command to vote on proposal >> `vega wallet command --name="testing-mainnet" --pubkey="<my-public-key>" '{"proposalSubmission": {"reference": "some-ref", "terms": {"closingTimestamp": "1234567890", "enactmentTimestamp": "1234567891", "validationTimestamp": "1234567892", "updateNetworkParameter": { "changes": { "key": "<network-parameter>", "value": "<new-value>" } } } } }'`
+
+**NOTE:** This action will only work if the parameter/config is able to be changed via a governance proposal AND time is not critical in the parameter being updated. If this is time critical it's recommended to jump to go to Scenario 2.4. 
```

Comment:
@JonRay15 
Non critical change... Can this link to the section below?

Reply:
This is done